### PR TITLE
Cleanup installer plugins

### PR DESCRIPTION
**Pull request by roland-d**
#### Summary of Changes

This change does a few things:
- Split the URL/Folder installation plugin into two separate plugins
- Removed obsolete files
- Removed obsolete code
- Cleanup other issues (thank you @andrepereiradasilva taken also from #10296)
- Added a message if no plugin is enabled
- Moved the language strings to the plugin language files
#### Testing Instructions for a clean installation
1. Download the zip file from this pull request containing the complete Joomla installation
2. Do a clean installation
3. Check the tabs are still under Manage -> Install
#### Testing Instructions for an update
1. Install Joomla 3.5.1
2. Download the zip file from this pull request containing the complete Joomla installation
3. Setup a local update server
4. Update the 3.5.1 to 3.6.0 using the local update server
5. Check the tabs are still under Manage -> Install
### Testing Instructions for both
1. Disable all installer plugins in Plugin Manager
2. Go to Manage -> Install and check there is a message saying there are no plugins enabled
3. Check the link in the message works

### CODE REVIEW

File: administrator/components/com_admin/sql/updates/postgresql/3.6.0-2016-05-06.sql [hunk 1]
```diff
@@ -1 +1,5 @@
-DELETE FROM "#__extensions" WHERE "type" = "library" AND "element" = "simplepie";
\ No newline at end of file
+DELETE FROM "#__extensions" WHERE "type" = "library" AND "element" = "simplepie";
+INSERT INTO `#__extensions` (`extension_id`, `name`, `type`, `element`, `folder`, `client_id`, `enabled`, `access`, `protected`, `manifest_cache`, `params`, `custom_data`, `system_data`, `checked_out`, `checked_out_time`, `ordering`, `state`) VALUES
+(455, 'plg_installer_packageinstaller', 'plugin', 'packageinstaller', 'installer', 0, 1, 1, 1, '', '', '', '', 0, '0000-00-00 00:00:00', 1, 0),
+(456, 'plg_installer_folderinstaller', 'plugin', 'folderinstaller', 'installer', 0, 1, 1, 1, '', '', '', '', 0, '0000-00-00 00:00:00', 2, 0),
```
Comment (andrepereiradasilva) [inline administrator/components/com_admin/sql/updates/postgresql/3.6.0-2016-05-06.sql]: the tables and fields quoteName are not correct in postgresql. they should be `"` (see the other ones)
Also the dates should be `1970-01-01 00:00:00`, not `0000-00-00 00:00:00`

File: installation/sql/postgresql/joomla.sql [hunk 2]
```diff
@@ -611,8 +611,9 @@ INSERT INTO "#__extensions" ("extension_id", "name", "type", "element", "folder"
...
-(455, 'plg_installer_packageinstaller', 'plugin', 'packageinstaller', 'installer', 0, 1, 1, 1, '{"name":"plg_installer_packageinstaller","type":"plugin","creationDate":"07.05.2016","author":"Joomla","copyright":"Copyright (C) 2016 Open Source Matters. All rights reserved.","authorEmail":"admin@joomla.org","authorUrl":"www.joomla.org","version":"1.0","description":"PLG_PACKAGE_INSTALLER_PLUGIN_XML_DESCRIPTION","group":"","filename":"packageinstaller"}', '{}', '', '', 0, '0000-00-00 00:00:00', 1, 0),
-(456, 'plg_installer_urlfolderinstaller', 'plugin', 'urlfolderinstaller', 'installer', 0, 1, 1, 1, '{"name":"plg_installer_urlfolderinstaller","type":"plugin","creationDate":"07.05.2016","author":"Joomla","copyright":"Copyright (C) 2016 Open Source Matters. All rights reserved.","authorEmail":"admin@joomla.org","authorUrl":"www.joomla.org","version":"1.0","description":"PLG_INSTALLER_PLUGIN_XML_DESCRIPTION","group":"","filename":"urlfolderinstaller"}', '{}', '', '', 0, '0000-00-00 00:00:00', 0, 0);
+(455, 'plg_installer_packageinstaller', 'plugin', 'packageinstaller', 'installer', 0, 1, 1, 1, '', '', '', '', 0, '0000-00-00 00:00:00', 1, 0),
```
Comment (andrepereiradasilva) [inline installation/sql/postgresql/joomla.sql]: the dates should be 1970-01-01 00:00:00, not 0000-00-00 00:00:00

File: plugins/installer/folderinstaller/folderinstaller.php [hunk 3]
```diff
@@ -0,0 +1,95 @@
...
+{
+	/**
+	 * Constructor
+	 *
+	 * @param   object  &$subject  The object to observe
+	 * @param   array   $config    An optional associative array of configuration settings.
+	 *                             Recognized key values include 'name', 'group', 'params', 'language'
+	 *                             (this list is not meant to be comprehensive).
+	 *
+	 * @since   1.5
```
Comment (andrepereiradasilva) [inline plugins/installer/folderinstaller/folderinstaller.php]: 1.5? or 3.6.0

**Comment by zero-24** [comment]
Download of the brach is: https://github.com/roland-d/joomla-cms/archive/cleanup-installer-plugins.zip

**Comment by zero-24** [comment]
I have tesed the clean install and disable the plugins both works!

I have not got the update server runnung i will try it later.

**Comment by alikon** [comment]
I have tested this item :white_check_mark: successfully on 8b9add867a16d8816ba50c26a08d57923b420832<br /><br />tested on a clean installation<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/10308">issues.joomla.org/joomla-cms/10308</a>.</sub>

**Comment by brianteeman** [comment]
RTC<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/10308">issues.joomla.org/joomla-cms/10308</a>.</sub>

**Comment by Kubik-Rubik** [comment]
Thank you @roland-d and testers!

**Comment by zero-24** [comment]
please remove the label @joomla-cms-bot

**Pull request merged**